t/11-tcp-open.t

```diff
@@ -37,6 +37,23 @@ sub socketpair_inet
     return ($rd, $wr);
 }
 
+# Launch watchdog on Windows in background
+if( $^O eq 'MSWin32' ) {
+    my $child= system(1, $^X,'-e',"sleep 5; kill 9 => $$");
```

Comment:
For clarity here I'd write:

```perl
my $ppid = $$;
my $child = system(1, $^X,'-e',"sleep 5; kill 9 => $ppid");
```

just to make it clearer that the invoked process will be trying to kill its parent, and not itself. Also can we put KILL instead of 9?